web/src/app/settings/settings-group/settings-group.component.sass

```diff
@@ -0,0 +1,49 @@
+$triangle-height: 30px
+$persistent-color: #D4E157
+$periodical-color: #FFCA28
+$occasional-color: #FFA726
+
+.group-header
+  margin-top: -$triangle-height
+  .title
+    display: inline-block
+    padding: 10px 0 0 15px
+
+.triangle-prefix
+  width: 0
+  border-style: solid
+  border-width: $triangle-height 25px 0 0
+
+  &.persistent
+    border-color: $persistent-color transparent transparent transparent
+
+  &.periodical
+    border-color: $periodical-color transparent transparent transparent
+
+  &.occasional
+    border-color: $occasional-color transparent transparent transparent
```

Comment:
It is very convenient way to use mixins in cases when we need to duplicate styles